README.rst

```diff
@@ -106,27 +119,30 @@ the PSF for inclusion in the documentation.
 
 接著在 terminal 裡按照以下步驟：
 
-1. 基於最新版本的 ``upstream/3.9`` 開啟一個 branch，現在假設我們想要翻譯 Glossary
+1. 基於最新版本的 ``upstream/3.9`` 開啟一個 branch，現在假設我們想要翻譯 Glossary \
    所以把這個 branch 叫做 ``glossary`` ::
 
     git fetch upstream
     git checkout -b glossary upstream/3.9
 
-2. 接著就可以開始翻譯，你可以使用 Poedit 應用程式將檔案打開，或是用以下指令\
-   （翻譯不同檔案時將 glossary 換成別的檔名） ::
+2. 接著就可以開始翻譯，你可以手動開啟 Poedit 應用程式再選檔案或用以下指令請 Poedit 將檔案\
+   打開，翻譯不同檔案時將 glossary 換成別的檔名） ::
 
     poedit glossary.po
 
-3. 存檔以後，可以輸出文件以確保你的修改沒有 rST 的語法錯誤或警告 ::
+3. 存檔以後，執行以下列指令編譯輸出文件，以確保你的修改沒有 rST 的語法錯誤或警告 ::
```

Comment:
Just found out something worth mentioned here (I can open another PR for this so don't worry):

Actually the Makefile expected the translators to do the translation on `3.X` branch here and `make` command would fail if run on a branch that CPython does not have a corresponding one. I [did a hack](https://github.com/python/python-docs-zh-tw/pull/102#discussion_r733340801) when I implementing GitHub Action (but forgot to edit README). It allows translators to run the command below by specifying an argument `VERSION`.

```
VERSION=3.9 make
```